draft-ietf-quic-transport.md

```diff
@@ -678,6 +687,34 @@ packet protection in detail.  After decryption, the plaintext consists of a
 sequence of frames, as described in {{frames}}.
 
 
+## Compound Packets {#packet-compound}
+
+Compound packets allow the sender to assemble an Initial ({{packet-initial}})
+or a Handshake ({{packet-handshake}}) packet and a Protected
+({{packet-protected}}) packet into one UDP packet, thereby reducing the number
+of packets required to be emitted when application data can be sent during the
+handshake.
```

Comment:
Question, and important to discuss either way:  When sending Compound Initial + 0-RTT, does the minimum packet size apply to the entire compound and not just the Initial section?  If so, might be worth mentioning that one intended use-case of this is to meet the minimum with 0-RTT data instead of PADDING.